app/src/main/java/com/stevesoltys/seedvault/restore/RestoreFilesFragment.kt

```diff
@@ -34,7 +35,9 @@ internal class RestoreFilesFragment : SnapshotFragment() {
         val skipView: TextView = footer.findViewById(R.id.skipView)
         skipView.setOnClickListener {
             requireActivity().apply {
-                setResult(RESULT_OK)
+                // Equivalent to com.google.android.setupcompat.util.ResultCodes.RESULT_SKIP
+                // SetupWizard handles this
+                setResult(RESULT_FIRST_USER)
                 finishAfterTransition()
```

Comment:
I don't really remember why I was setting `RESULT_OK` here before. Looks like Seedvault itself isn't caring about the result, so I can only think of SUW in which case the new result should be better. Or how does SUW treat result OK? Move to the next step as well?

Asking, because there's also `RestoreFilesStartedFragment` below in the same file which sets a result.

Reply:
> Or how does SUW treat result OK? Move to the next step as well?

Yes but for that it does a transition I believe.

https://review.lineageos.org/c/LineageOS/android_packages_apps_SetupWizard/+/315577/2/src/org/lineageos/setupwizard/backup/RestoreIntroActivity.java

This handles RESULT_SKIP

RESULT_OK would go to https://github.com/LineageOS/android_packages_apps_SetupWizard/blob/lineage-19.0/src/org/lineageos/setupwizard/SubBaseActivity.java#L129


I only tested this change with skipping so far, not with a full restore.